src/App.tsx

```diff
@@ -1,61 +1,100 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
 
 export const App = () => {
+  const [text, setText] = useState('');
+  const [user, setUser] = useState(0);
```

Comment:
I suppose here should be the userId instead
```suggestion
  const [userId, setUserId] = useState(0);
```